pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -73,7 +72,6 @@
 }:
 
 assert stdenv.cc.libc or null != null;
-assert pipewireSupport -> !waylandSupport || !webrtcSupport -> throw "pipewireSupport requires both wayland and webrtc support.";
```

Comment:
Please do not just remove it. Disable it for now with a comment why we are disabling it and linking to this PR. I am very confident we will add it again in a matter of days/weeks. This will likely become a recurring theme as long as pipewire isn't stable. We have a lot of pipewire users and if those want to enable it again it shouldn't be reinventing everything again. :)